docs/farming-&-staking/farming/additional-guides/network-info.md

```diff
@@ -6,72 +6,117 @@ slug: networking
 keywords:
     - Farmer
     - Farming
-    - CLI
-    - Binaries
+    - Node
+    - Domain
+    - RPC
     - Port Forwarding
-    - Autonomys Desktop
     - Ports
     - Networking
     - Network
 ---
 
-:::caution
-If you are using Linux and enabled `ufw` (firewall) ensure that your firewall allows traffic on the necessary ports: `sudo ufw allow 30333,30433 comment 'Autonomys Node'` for the node and `sudo ufw allow 30533 comment 'Autonomys Farmer'` for the farmer. Adjust the port numbers if you have changed them from the defaults in your startup parameters. Also, if you are connecting remote farmers to a node and your firewall rules do not permit devices in your LAN to communicate freely, you will need to add a rule to your node server's firewall (assuming your LAN IP range is 192.168.1.0/24 in this example):  `sudo ufw allow from 192.168.1.0/24 to any port 9944 proto tcp comment 'Remote Farmers RPC'`.
-:::
-
-Configuring the right network settings is crucial for the effective operation of blockchain nodes. This guide provides detailed information about the necessary ports, network configurations, and steps for port forwarding for Autonomys Network products, ensuring smooth communication and optimal performance.
+## Overview
 
-## Understanding Port Forwarding
+This guide explains how to configure network settings and set up port forwarding for Autonomys Network. Correct port forwarding is essential to ensure proper communication between your node, farmer, and the Autonomys Network. 
 
-Forwarding ports on your router will open up a specific lane on your router to allow external traffic to flow in or out. This process is usually automatic for common applications, but for specific needs like ours, it must be done manually. For a foundational understanding of port forwarding, you can read this guide: [Port Forwarding Overview](https://www.geeksforgeeks.org/port-forwarding-on-router-and-why-do-we-need-it/).
+By opening specific ports on your router, you allow external traffic to reach your node, which is necessary for blockchain synchronization, data exchange, and remote procedure calls (RPC).
 
 ## Required Ports for Autonomys Network
 
-For optimal performance of our blockchain node, specific ports need to be configured to handle incoming connections. These ports facilitate various critical functions within our blockchain network
+To ensure seamless communication within the P2P protocol, specific network ports must remain open to facilitate data transmission. These ports support essential blockchain operations, including block synchronization, transaction propagation, and Remote Procedure Call (RPC) interactions.
```

Comment:
I believe we can omit this sentence, as it repeats the information from the last paragraph